pages/getting-started/install-memgraph/docker.mdx

```diff
@@ -165,14 +165,14 @@ To load this image into Docker, do the following:
 {<h3>Uncompress the downloaded file</h3>}
 
 The downloaded `.tar.gz` file needs to be uncompressed so that you get file
-named `memgraph-2.13.0-docker.tar`. Do not unpack the `.tar` archive. This
+named `memgraph-2.17.0-docker.tar.gz`. Do not unpack the `.tar` archive. This
 process varies depending on your operating system:
 
 - **Windows**: Use a tool like 7-Zip to extract the contents of the `.tar.gz` file
 - **macOS/Linux**: Use the terminal with the following command:
```

Comment:
Why do we have a good explanation for the Linux CLI, but we don't have anything specific about 7zip for Windows? No screenshot, no copy/paste command.

Reply:
Yes, that's the right idea. It would be nice to include instructions such as "right click on the icon" or whatever so the user can follow and complete their task.

Reply:
I'll prepare the images and text.